src/app/components/fileattachments/index.html

```diff
@@ -1,25 +1,213 @@
+          </div>
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+      
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+            <p>Use file attachment when the user needs to add metadata to the uploaded file.</p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <div>
+            <img src="~/assets/img/guidelines/file-attach/use-metadata.png" />
+            <div class="caption caption-success sky-text-success">Do use file attachment to add metadata about the uploaded file.</div>
+          </div>
+        </sky-docs-visual-example-image>
+      </sky-docs-visual-example>
+
+    <h3 class="sky-text-danger">Don't use when</h3>
+      <p class="sky-text-block">
+        Don't use file attachment if the user needs to upload one local file. Use single file attachment instead.
+      </p>
+  </sky-demo-page-content>
+
+    <sky-demo-page-content sectionHeading="Anatomy">
+      <sky-docs-anatomy imagePath="~/assets/img/guidelines/file-attach/anatomy.png">
+        <sky-docs-anatomy-item anatomyNumber="1">Browse or drop local file</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="2">File name</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="3">File size</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="4">File preview</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="5">Delete button</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="6" isOptional="true">Attach from a URL</sky-docs-anatomy-item>
+        <sky-docs-anatomy-item anatomyNumber="7" isOptional="true">Additional metadata</sky-docs-anatomy-item>
+      </sky-docs-anatomy>
+    </sky-demo-page-content>
+    
+    <sky-demo-page-content sectionHeading="Options">
+      <h3>Attach from URL</h3>
+        <p class="sky-text-block">
+          You can specify whether you want the user to be able to attach a file from a URL or only from the local device.
+        </p>
+      <h3>Validation</h3>
+        <p class="sky-text-block">
+          You can specify the parameters for a valid file, including file type, file size or a custom validation function.
+        </p>
+    </sky-demo-page-content>
+    
+    <sky-demo-page-content sectionHeading="Behavior & states">
+      <h3>File information</h3>
+      <sky-docs-visual-example>
+        <sky-docs-visual-example-text>
+            <p>When the user attaches a file, summary information for that file appears in a card below the file attach input. By default the summary information includes file name, a delete button, and if the file is from a local drive also includes file size and a file thumbnail. You can optionally include inputs for the user to add metadata about the file.</p>
+        </sky-docs-visual-example-text>
+        <sky-docs-visual-example-image>
+          <img src="~/assets/img/guidelines/file-attach/behavior-preview.png" />
```

Comment:
Why does this image user borders when the other don't?

Reply:
to more clearly demarcate the before/after states, since the arrow is +/- the same color as a lot of the UI it seemed confusing without the borders.

Reply:
Sounds good. Thanks.